docs/platforms/unity/native-support/index.mdx

```diff
@@ -74,3 +74,11 @@ If you don't want to rely on the automated symbol upload, you can run `sentry-cl
 ```bash
 sentry-cli --auth-token ___ORG_AUTH_TOKEN___ debug-files upload --org ___ORG_SLUG___ --project ___PROJECT_SLUG___ PATH_TO_SYMBOLS
 ```
+
+## Store Minidumps as Attachments
+
+<Include name="store-minidumps-as-attachments-intro" />
+
+<Alert level="note">☝ This feature is currently available only for iOS and Android devices.</Alert>
+
+<Include name="store-minidumps-as-attachments-configuration" />
```

Comment:
I wish I could use a React component like `<Include name="store-minidumps-as-attachments-configuration" includeOSNote />`. However, since we're only using `.mdx` for this, I decided to split it up instead.